### PR TITLE
Send to Compliance

### PR DESCRIPTION
**Pull request by ghickman**
This adds a Compliance Settings menu item in the Wagtail admin to set a global email address to send notifications to Compliance with.  It also sends an email to compliance when a Project is approved, using the eamil address in this setting.

### CODE REVIEW

**Comment by ghickman** [comment]
When a Compliance email hasn't been set this silently does nothing, should we do something a little louder about this?

File: opentech/apply/projects/views.py [hunk 1]
```diff
@@ -162,9 +184,11 @@ def form_valid(self, form):
             source=project,
         )
 
+        project.send_to_compliance(self.request)
+
         project.is_locked = False
         project.status = CONTRACTING
-        project.save(update_fields=['is_locked', 'status'])
+        project.save(update_fields=['is_locked', 'sent_to_compliance_at', 'status'])
```
Comment (todd-dembrey) [inline opentech/apply/projects/views.py]: `sent_to_compliance_at` is handled in the `send_to_compliance` method.

File: opentech/public/utils/models.py [hunk 2]
```diff
@@ -246,6 +246,11 @@ class Meta:
     ]
 
 
+@register_setting
+class ComplianceSettings(BaseSetting):
```
Comment (todd-dembrey) [inline opentech/public/utils/models.py]: This isnt part of the public site. It should exist under apply.

Also - we probably want to call it `ProjectSettings` so we can add the finance email as well.

File: opentech/apply/projects/models.py [hunk 3]
```diff
@@ -184,6 +186,8 @@ class Project(models.Model):
     )
     created_at = models.DateTimeField(auto_now_add=True)
 
+    sent_to_compliance_at = models.DateTimeField(null=True)
```
Comment (todd-dembrey) [inline opentech/apply/projects/models.py]: Missing from simplified project page

>  Date of sending is tracked on the model and added to the simplified project page

File: opentech/apply/activity/templates/messages/email/sent_to_compliance.html [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{% extends "messages/email/base.html" %}
+
+{% block salutation %}{% endblock %}
+
+{% block content %}
+A Project has been approved:
+
+Title: {{ source.title }}
+Link: {{ request.scheme }}://{{ request.get_host }}{% url 'apply:projects:detail' pk=source.pk %}
```
Comment (todd-dembrey) [inline opentech/apply/activity/templates/messages/email/sent_to_compliance.html]: missing:
```
Lead contact details is included in the email
Simplified views included in email - #1399, #1372
```

```
A project is awaiting your review. 

Title: {{ source.title }}
Please click here to view it: <path to simplified view>

Please contract {{ source.lead }} - {{ source.lead.email }} if you have any questions
```

File: opentech/apply/projects/views.py (hunk 1 above)
Comment (ghickman) [reply opentech/apply/projects/views.py]: Fixed and rebased.

File: opentech/public/utils/models.py (hunk 2 above)
Comment (ghickman) [reply opentech/public/utils/models.py]: Fixed and rebased.

File: opentech/apply/projects/models.py (hunk 3 above)
Comment (ghickman) [reply opentech/apply/projects/models.py]: Fixed and rebased.

File: opentech/apply/activity/templates/messages/email/sent_to_compliance.html (hunk 4 above)
Comment (ghickman) [reply opentech/apply/activity/templates/messages/email/sent_to_compliance.html]: Fixed and rebased.

**Pull request merged**